extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -3329,4 +3329,75 @@ static public function prefetchWikisById( $ids, $what = self::PREFETCH_DEFAULT )
 		$wgMemc->prefetch($keys);
 	}
 
+	/**
+	 * Renders community's value of given variable
+	 *
+	 * @access public
+	 * @static
+	 *
+	 * @param string $name name of wg variable
+	 * @param string $type type of variable ($variable->cv_variable_type)
+	 *
+	 * @return string
+	 */
+	static public function renderValueOnCommunity( $name, $type ) {
+		global $$name;
+		global $preWFValues;
+
+		$value = "";
+
+		if( isset( $preWFValues[$name] ) ) {
+			// was modified, spit out saved default
+			$value = self::parseValue( $preWFValues[$name], $type );
+		} elseif( isset( $$name ) ) {
+			// was not modified, spit out actual value
+			$value = self::parseValue( $$name, $type );
+		}
+		return htmlspecialchars( $value );
+	}
+
+	/**
+	 * Renders wg variable
+	 *
+	 * @access public
+	 * @static
+	 *
+	 * @param object $variable parameter passed to variable.tmpl.php
+	 *
+	 * @return string
+	 */
+	static public function renderValue( $variable ) {
```

Comment:
getValue or getEscapedValue, I personally wouldn't put htmlspecialchars in there (but in template perhaps, even if it would have to be there few times).